tests/utils.go

```diff
@@ -1737,8 +1737,11 @@ func removeNamespaces() {
 	fmt.Println("")
 	for _, namespace := range testNamespaces {
 		fmt.Printf("Waiting for namespace %s to be removed, this can take a while ...\n", namespace)
-		EventuallyWithOffset(1, func() bool { return errors.IsNotFound(virtCli.CoreV1().Namespaces().Delete(namespace, nil)) }, 240*time.Second, 1*time.Second).
-			Should(BeTrue())
+		EventuallyWithOffset(1, func() bool {
+			err = (virtCli.CoreV1().Namespaces().Delete(namespace, nil))
+			return errors.IsNotFound(err)
+		}, 240*time.Second, 1*time.Second).
+			Should(BeTrue(), "While deleting test namespace %s an unexpected error occurred : %v", namespace, err)
```

Comment:
I think you should return the error and then evaluate it instead the boolean.
I would expect the error to include the namespace and other data about why it failed. Could you check how this assertion below outputs the failure? I would expect it to print the returned value (the error) so some of the additional message you added may not be needed.

So it should look like this:
```
EventuallyWithOffset(1, func() error {
	return virtCli.CoreV1().Namespaces().Delete(namespace, nil)
	return errors.IsNotFound(err)
}, 240*time.Second, 1*time.Second).
	Should(WithTransform(errors.IsNotFound, BeTrue()), "While deleting test namespace %s an unexpected error occurred : %v", namespace, err)
```

Reply:
> I think you should return the error and then evaluate it instead the boolean.
> I would expect the error to include the namespace and other data about why it failed. Could you check how this assertion below outputs the failure? I would expect it to print the returned value (the error) so some of the additional message you added may not be needed.
> 
> So it should look like this:
> 
> ```
> EventuallyWithOffset(1, func() error {
> 	return virtCli.CoreV1().Namespaces().Delete(namespace, nil)
> 	return errors.IsNotFound(err)
I guess this line is mistake?
> }, 240*time.Second, 1*time.Second).
> 	Should(WithTransform(errors.IsNotFound, BeTrue()), "While deleting test namespace %s an unexpected error occurred : %v", namespace, err)
> ```

Thanks @EdDev. the error output includes the object type we tried to delete (namesapce) and the objects name. so I updated the additional message. 
the snippet as it is won't work in case of a `nil` error, so I added a condition to verify that.